tests/e2e/parallel/main_test.go

```diff
@@ -2795,3 +2797,189 @@ func TestScheduledSuiteTimeoutFail(t *testing.T) {
+		t.Fatal(err)
+	}
+	defer f.Client.Delete(context.TODO(), &scanSettingBinding)
+
+	// Wait until the first scan completes since the CronJob is created
+	// after the scan is done
+	if err := f.WaitForSuiteScansStatus(f.OperatorNamespace, bindingName, compv1alpha1.PhaseDone, compv1alpha1.ResultNonCompliant); err != nil {
+		t.Fatal(err)
+	}
+
+	// Assert the ComplianceSuite.suspend attribute is False
+	suite := &compv1alpha1.ComplianceSuite{}
+	key := types.NamespacedName{Name: bindingName, Namespace: f.OperatorNamespace}
+	if err := f.Client.Get(context.TODO(), key, suite); err != nil {
+		t.Fatal(err)
+	}
+	if suite.Spec.Suspend == true {
+		t.Fatalf("ComplianceSuite %s is suspended when it should not be", suite.Name)
+	}
+
+	// Assert the CronJob associated with the ComplianceSuite is False
+	job := &batchv1.CronJob{}
+	jobName := compsuitectrl.GetRerunnerName(suite.Name)
+	err := f.WaitForObjectToExist(jobName, f.OperatorNamespace, job)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	if *job.Spec.Suspend == true {
+		t.Fatalf("CronJob %s is suspended when it should not be", jobName)
+	}
+
+	// Suspend the `ScanSetting` using the `suspend` attribute
+	scanSettingUpdate := &compv1alpha1.ScanSetting{}
+	if err := f.Client.Get(context.TODO(), types.NamespacedName{Namespace: f.OperatorNamespace, Name: scanSettingName}, scanSettingUpdate); err != nil {
+		t.Fatalf("failed to get ScanSetting %s", scanSettingName)
+	}
+	scanSettingUpdate.Suspend = true
+	if err := f.Client.Update(context.TODO(), scanSettingUpdate); err != nil {
+		t.Fatal(err)
+	}
+
+	// Assert the `CronJob` associated with the `ComplianceSuite` is set to `suspend=true`
+	job = &batchv1.CronJob{}
+	jobName = compsuitectrl.GetRerunnerName(suite.Name)
+	err = f.Client.Get(context.TODO(), types.NamespacedName{Name: jobName, Namespace: f.OperatorNamespace}, job)
+	if err != nil {
+		t.Fatal(err)
+	}
+	if *job.Spec.Suspend == false {
+		t.Fatalf("Expected CronJob %s to be suspended", jobName)
+	}
+
+	// Resume the `ComplianceScan` by updating the `ScanSetting.suspend` attribute to `False`
+	scanSettingUpdate = &compv1alpha1.ScanSetting{}
+	if err := f.Client.Get(context.TODO(), types.NamespacedName{Namespace: f.OperatorNamespace, Name: scanSettingName}, scanSettingUpdate); err != nil {
+		t.Fatalf("failed to get ScanSetting %s", scanSettingName)
+	}
+	scanSettingUpdate.Suspend = false
+	if err := f.Client.Update(context.TODO(), scanSettingUpdate); err != nil {
+		t.Fatal(err)
+	}
+
+	// Assert the `CronJob` associated with the `ComplianceScan` is set to `suspend=false`
+	job = &batchv1.CronJob{}
+	jobName = compsuitectrl.GetRerunnerName(suite.Name)
+	err = f.Client.Get(context.TODO(), types.NamespacedName{Name: jobName, Namespace: f.OperatorNamespace}, job)
+	if err != nil {
+		t.Fatal(err)
+	}
+	if *job.Spec.Suspend == true {
+		t.Fatalf("CronJob %s is suspended when it should not be", jobName)
+	}
+
+}
+
+func TestSuspendScanSettingDoesNotCreateScan(t *testing.T) {
```

Comment:
Maybe we should move these to serial tests since they will fail if they're run at the same time as another test that's using `ocp4-cis` or `ocp4-cis-node`.